tests/bits/count_dofs_per_component_hp_01.cc

```diff
@@ -50,33 +47,19 @@ test()
 
   FESystem<dim> fe_system(u, 2, p, 1);
 
-  hp::FECollection<dim> fe_collection;
-  fe_collection.push_back(fe_system);
-
-  hp::DoFHandler<dim> hp_dof_handler(triangulation);
-  DoFHandler<dim>     dof_handler(triangulation);
+  DoFHandler<dim> dof_handler(triangulation);
```

Comment:
Were the changes in this test intended? We previously checked the function both for the hp and the non-hp case, but yo're now only checking the non-hp case.

Reply:
Yes, they were intended. We now decide between hp and non-hp mode depending on the amount of finite elements in the `FECollection`: if it is only one element, non-hp mode is invoked, otherwise we're in hp mode. The `DoFHandler` stores any finite element in a `FECollection` internally.

So in this example, both objects `dof_handler` and `hp_dof_handler` are in fact operating in non-hp mode since `fe_collection` only contains one finite element. As both `DoFHandlers` are identical, I removed one of them.

Reply:
Ah, I had missed that there is only one element in the collection.